selenium_tests/masquerade/canvas_masquerade_page_object.py

```diff
@@ -0,0 +1,33 @@
+from selenium.webdriver.common.by import By
+from selenium_tests.lti_emailer.page_objects.emailer_base_page_object import EmailerBasePageObject
+from urlparse import urljoin
+
+
+class CanvasMasqueradePageLocators(object):
+    CONFIRM_BUTTON = (By.CSS_SELECTOR, "a.btn.masquerade_button")
+    MASQUERADE_URL = 'users/{}/masquerade'
+
+
+class CanvasMasqueradePageObject(EmailerBasePageObject):
```

Comment:
Is there some reason this isn't inheriting from the `BasePageObject`, or the `CanvasBasePageObject`? It is outside of the lti_emailer tree structure, so I wouldn't expect it to be dependent upon any of lti_emailer's page objects.